tdrs-backend/tdpservice/parsers/parse.py

```diff
@@ -62,6 +61,35 @@ def parse_datafile(datafile):
 
     errors = errors | line_errors
 
+    # errors['summary'] = DataFileSummary.objects.create(
```

Comment:
am I understanding correctly that a `DataFileSummary` is not yet created when the parser task kicks off?
<img width="276" alt="image" src="https://github.com/raft-tech/TANF-app/assets/111305129/0e41d14d-a73c-4ec9-995f-dfaf7e50dab3">

Reply:
Yes, in the second PR we ended up putting this invocation in `scheduling/parser_task.py`